### PR TITLE
Config Users - blue bar with Fieldset Description

### PR DESCRIPTION
**Pull request by pe7er**
Most **Component Configuration Options** have under the option tab **a blue bar with a description**
### Summary of Changes

This PR adds a blue bar with description to **User Options**, **Password Options** and **User Notes History** to make it consistent with other options & components.
#### Before the PR:

backend: System > Global Configuration > (in Left menu) Users > [**User Options**]

info-label-options-users-useroptions-before

backend: System > Global Configuration > (in Left menu) Users > [**Password Options**]

info-label-options-users-passwordoptions-before

backend: System > Global Configuration > (in Left menu) Users > [**User Notes History**]

info-label-options-users-usernoteshistory-before
#### After the PR:

backend: System > Global Configuration > (in Left menu) Users > [**User Options**]

info-label-options-users-useroptions-after

backend: System > Global Configuration > (in Left menu) Users > [**Password Options**]

info-label-options-users-passwordoptions-after

backend: System > Global Configuration > (in Left menu) Users > [**User Notes History**]

info-label-options-users-usernoteshistory-after

### CODE REVIEW

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on 0deed5e017bfd817772868726f8dcafa9d68d033<br /><br />works :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12087">issues.joomla.org/joomla-cms/12087</a>.</sub>

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on 0deed5e017bfd817772868726f8dcafa9d68d033<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12087">issues.joomla.org/joomla-cms/12087</a>.</sub>

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on 0deed5e017bfd817772868726f8dcafa9d68d033<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12087">issues.joomla.org/joomla-cms/12087</a>.</sub>

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 0deed5e017bfd817772868726f8dcafa9d68d033<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12087">issues.joomla.org/joomla-cms/12087</a>.</sub>

**Comment by jeckodevelopment** [comment]
@brianteeman we need you also here!

File: administrator/language/en-GB/en-GB.com_users.ini [hunk 1]
```diff
@@ -59,6 +59,7 @@ COM_USERS_CONFIG_FIELD_MINIMUM_UPPERCASE_DESC="Set the minimum number of upper c
 COM_USERS_CONFIG_FIELD_NEW_USER_TYPE_DESC="The default group that will be applied to New Users Registering via the Frontend."
 COM_USERS_CONFIG_FIELD_NEW_USER_TYPE_LABEL="New User Registration Group"
 COM_USERS_CONFIG_FIELD_NOTES_HISTORY="User Notes History"
+COM_USERS_CONFIG_FIELD_NOTES_HISTORY_FIELDSET_DESC="Configure the versioning settings for user notes."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_users.ini]: User Notes need to be capitalised

File: administrator/language/en-GB/en-GB.com_users.ini [hunk 2]
```diff
@@ -69,8 +70,10 @@ COM_USERS_CONFIG_FIELD_USERACTIVATION_OPTION_ADMINACTIVATION="Administrator"
 COM_USERS_CONFIG_FIELD_USERACTIVATION_OPTION_SELFACTIVATION="Self"
 COM_USERS_CONFIG_IMPORT_FAILED="An error was encountered while importing the configuration: %s."
 COM_USERS_CONFIG_PASSWORD_OPTIONS="Password Options"
+COM_USERS_CONFIG_PASSWORD_OPTIONS_FIELDSET_DESC="Configure the password complexity for your users to increase security."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_users.ini]: Dont introduce a new term (complexity) that doesnt exist elsewhere - just stick with options

**Comment by brianteeman** [comment]
@jeckodevelopment commented inline - this PR is not correct

**Comment by jeckodevelopment** [comment]
Thanks

**Comment by brianteeman** [comment]
The blue bar is really not needed - you just added some meaningless text in order to justify adding th blue bar - my 2c

**Comment by pe7er** [comment]
I understand your point but if every **Options tab starts with a blue bar with some information**, **the UI for Options** will be **consistently the same on all Option Tabs**. That way an admin has an extra visual pointer (besides the "Options" in the Page Title) that they are on the Options page.

**Comment by brianteeman** [comment]
I don't agree in adding help text if it has no meaning or value. That has
the effect of users just ignoring it. It should only exist if it is
required information

**Comment by hans2103** [comment]
I agree with @BrianTeeman. If the information bar doesn't add any value then it should not be added. It should not be added just to be consistently. 
It might be better to change the title of the tabs which do need the information bar. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12087">issues.joomla.org/tracker/joomla-cms/12087</a>.</sub>

**Comment by brianteeman** [comment]
Closing for the reasons stated above

**Pull request closed without merging**